integration-test/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitSpecificIntegrationTestBase.java

```diff
@@ -3020,6 +3020,52 @@ public void scanAll_ScanAllGivenForPreparedWhenCoordinatorStateCommitted_ShouldR
         scanAll);
   }
 
+  @Test
+  public void get_GetWithMatchedConjunctionsGivenForCommittedRecord_ShouldReturnRecord()
+      throws TransactionException {
+    // Arrange
+    populateRecords(namespace1, TABLE_1);
+    DistributedTransaction transaction =
+        manager.begin(Isolation.SERIALIZABLE, SerializableStrategy.EXTRA_READ);
+    Get get =
+        Get.newBuilder(prepareGet(1, 1, namespace1, TABLE_1))
+            .where(ConditionBuilder.column(BALANCE).isEqualToInt(INITIAL_BALANCE))
+            .and(ConditionBuilder.column(SOME_COLUMN).isNullText())
+            .build();
+
+    // Act
+    Optional<Result> result = transaction.get(get);
+    transaction.commit();
+
+    // Assert
+    assertThat(result.isPresent()).isTrue();
+    assertThat(result.get().getInt(ACCOUNT_ID)).isEqualTo(1);
+    assertThat(result.get().getInt(ACCOUNT_TYPE)).isEqualTo(1);
+    assertThat(getBalance(result.get())).isEqualTo(INITIAL_BALANCE);
+    assertThat(result.get().getText(SOME_COLUMN)).isNull();
+  }
+
+  @Test
+  public void get_GetWithUnmatchedConjunctionsGivenForCommittedRecord_ShouldReturnEmpty()
+      throws TransactionException {
+    // Arrange
+    populateRecords(namespace1, TABLE_1);
+    DistributedTransaction transaction =
+        manager.begin(Isolation.SERIALIZABLE, SerializableStrategy.EXTRA_READ);
```

Comment:
I first created a bug regarding the extra read with get operations since I missed that integration tests are done with snapshot isolation by default, and this should be added in some cases.

Note that we might also be better to use `SERIALIZABLE` for some other tests (e.g., `get_GetGivenForCommittedRecord_ShouldReturnRecord`) because they are almost the same as the one in `DistributedTransactionIntegrationTestBase` if we run them with `SNAPSHOT`. However, I left it as is for now since it's a bit out of the scope of this PR.